order-processing-app/python_app/data/database.py

```diff
@@ -1,20 +1,19 @@
 from datetime import datetime, timedelta
 from pytz import utc
-from sqlalchemy import asc
 from flask_sqlalchemy import SQLAlchemy
-from sqlalchemy import or_, and_
+from sqlalchemy import asc, or_, and_, text
 import logging
 
-from flask_config import Config
+from python_app.flask_config import Config
 
 db = SQLAlchemy()
-from data.order import Order, PROCESSING, QUEUED, FAILED
+from python_app.data.order import Order, PROCESSING, QUEUED, FAILED
 
 
 def get_next_order_to_process() -> Order:
     # Find the next queued order and mark it as "Processing".
     updated_row_ids = list(
-        db.session.execute(
+        db.session.execute(text(
```

Comment:
This is the primary fix to the workshop. SQLAlchemy now throws a TextualSQLExpression error if the strings are not converted beforehand.